src/style.css

```diff
@@ -0,0 +1,80 @@
+body {
+  margin: 0;
+  --star-active: url("/images/star-active.svg");
+  --star: url("/images/star.svg");
+}
+
+.stars {
+  display: flex;
+  justify-content: center;
+}
+
+.stars__star {
+  background-image: var(--star);
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+  margin: 0 4px;
+  position: relative;
+}
+
+.stars--1 .stars__star:nth-child(1)::after {
+  content: "";
+  background-image: var(--star-active);
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars--2 .stars__star:nth-child(-n + 2)::after {
+  content: "";
+  background-image: var(--star-active);
+  position: absolute;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-repeat: no-repeat;
+  background-position: center;
+}
```

Comment:
why do you need to use after here, let's just set the background image depending on the pseudo-class